### ISSUE TITLE
regenie not detecting INFO from a pvar input

### ISSUE THREAD

**Issue by dvh13** [rgcgithub/regenie#551]
Hi,

Am running regenie_v3.4.1.gz_x86_64_Centos7_mkl
(havent persuaded v3.6 to run on our system yet, some library issue)

I am getting:

WARNING: Dosages are not present in the genotype file. Option --minINFO is skipped.

but INFO is in the pvar, I guess just not in the field format expected by regenie

my pvar, selected lines:

##INFO=<ID=INFO,Number=1,Type=Float,Description="IMPUTE2 info score">
#CHROM	POS	ID	REF	ALT	INFO
1	16587	1_16587_C_T	C	T	IMPUTED;MAF=0.000253911;AVG_CS=0.99981;R2=0.573077;AF=0.000253911;INFO=0.546338;F_MISSING=0;AN=102398;AC=26

thanks, david

**Comment by joellembatchou** [comment]
Hi David,

How did you generate the PLINK2 PGEN file? If it is from VCF, note that by default dosages are not imported (see [here](https://www.cog-genomics.org/plink/2.0/input#vcf)). Note that REGENIE can also handle BGEN format if you have it on hand.

Cheers,
Joelle

**Comment by dvh13** [comment]
Hi Joelle,

We made a plink2 file with 'dosage=HDS' for our GSA chip and TOPMed-r3 data.

The GWAS results for phenotypes tested were almost identical to the standard pgen conversion, at the cost of a much larger file and much much slower compute. We abandonded !

I will close this ticket.

d


On 8 Oct 2024, at 20:23, Joelle Mbatchou ***@***.***> wrote:



Hi David,

How did you generate the PLINK2 PGEN file? If it is from VCF, note that by default dosages are not imported (see here<https://www.cog-genomics.org/plink/2.0/input#vcf>). Note that REGENIE can also handle BGEN format if you have it on hand.

Cheers,
Joelle
